pkg/streamingpromql/planning/core/aggregate_expression_test.go

```diff
@@ -281,11 +281,11 @@ func TestAggregateExpression_Equivalence(t *testing.T) {
 
 	for name, testCase := range testCases {
 		t.Run(name, func(t *testing.T) {
-			require.Equal(t, testCase.expectEquivalent, testCase.a.EquivalentTo(testCase.b))
-			require.Equal(t, testCase.expectEquivalent, testCase.b.EquivalentTo(testCase.a))
+			require.Equal(t, testCase.expectEquivalent, testCase.a.EquivalentToIgnoringHintsAndChildren(testCase.b))
```

Comment:
Can we add a failure message here that includes the `a` and `b` expressions so that aren't getting an error message like "true is not false"?

Reply:
Good idea, added in ee3d1ae